src/main/java/com/example/demo/Carddata/Carddata.java

```diff
@@ -0,0 +1,36 @@
+package com.example.demo.Carddata;
+
+import jakarta.persistence.Entity;
+
+import jakarta.persistence.Table;
+
+@Entity
+@Table
+public class Carddata {
+    private Integer id;
+    private Double Card_num;
```

Comment:
Let's stick to camel case, I dont know this style

Reply:
Card number as Double??? Isn't card number something like this "1234 5678 9123 4567"

Reply:
done